docs/data-sources/aws_unity_catalog_assume_role_policy.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Deployment"
+---
+# databricks_aws_unity_catalog_assume_role_policy Data Source
+
+-> **Note** This resource has an evolving API, which may change in future versions of the provider. Please always consult [latest documentation](https://docs.databricks.com/data-governance/unity-catalog/get-started.html#configure-a-storage-bucket-and-iam-role-in-aws) in case of any questions.
+
+This data source constructs necessary AWS Unity Catalog assume role policy for you. [official documentation]
+
+## Example Usage
+
+```hcl
+data "databricks_aws_unity_catalog_policy" "this" {
+  aws_account_id = var.aws_account_id
+  bucket_name    = "databricks-bucket"
+  role_name      = "${var.prefix}-uc-access"
+  kms_name       = "databricks-kms"
+}
+
+data "databricks_aws_unity_catalog_assume_role_policy" "this" {
+  aws_account_id = var.aws_account_id
+  role_name      = "${var.prefix}-uc-access"
+  external_id    = "12345"
+}
+
+resource "aws_iam_policy" "unity_metastore" {
+  name   = "${var.prefix}-unity-catalog-metastore-access-iam-policy"
+  policy = data.databricks_aws_unity_catalog_policy.this.json
+}
+
+resource "aws_iam_role" "metastore_data_access" {
+  name                = "${var.prefix}-uc-access"
+  assume_role_policy  = data.aws_iam_policy_document.passrole_for_uc.json
+  managed_policy_arns = [aws_iam_policy.unity_metastore.arn]
+}
+```
+
+## Argument Reference
+
+* `aws_account_id` (Required) The Account ID of the current AWS account (not your Databricks account).
+* `external_id` (Required) The [storage credential](../resources/storage_credential.md) external id.
+* `role_name` (Required) The name of the AWS IAM role that you created in the previous step in the [official documentation](https://docs.databricks.com/data-governance/unity-catalog/get-started.html#configure-a-storage-bucket-and-iam-role-in-aws).
+* `kms_name` (Optional) If encryption is enabled, provide the name of the KMS key that encrypts the S3 bucket contents. If encryption is disabled, do not provide this argument.
```

Comment:
There is no such parameter in the struct

Reply:
removed